engine-tests/src/tests/erc20.rs

```diff
@@ -232,7 +235,10 @@ fn deploy_erc_20_out_of_gas() {
     test_utils::validate_address_balance_and_nonce(
         &runner,
         test_utils::address_from_secret_key(&source_account),
-        Wei::new_u64(INITIAL_BALANCE),
+        // Wei::new_u64(INITIAL_BALANCE),
+        Wei::new_u64(
+            INITIAL_BALANCE - (intrinsic_gas - 1) * GAS_PRICE - (intrinsic_gas + 1) * GAS_PRICE,
```

Comment:
No. Here should be something like this:
 ```suggestion
            INITIAL_BALANCE - error.gas_used * GAS_PRICE,
```